contribute/en/Style-and-Tone.md

```diff
@@ -83,11 +83,13 @@ Refer to UI labels in inverted commas (eg 'click on the “mute” button')
 
 Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open source”)
 
-“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together” (not “Mute a person” because one person might be using multiple channels).
+“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together”. 
 
-“Fader” The UI that controls a channel. “Each fader has a mute button”, “The person’s fader” ,“Group faders together” (not “The person’s channel” or “Mute a Fader”, not “Slider” or “Volume control”)
+“Fader” The UI used to control a channel. Not “slider” or “volume control”. Use when "channel" may not be appropriate, and preferebly only to refer to what a user sees on their screen. “Each fader has a mute button”, "leftmost fader". Remember: keep the user's view in mind when referring to the UI. For example, the visual effect of controlling a group of channels is to "make all the faders move together". But note that "fader" should **not** be used if "channel" would be a more accurate term. So, "reduce the channel volume" not "lower the fader". "Mute the channel" not "mute the fader". This is to disambiguate in cases where the UI isn't relevant, and help educate the reader in terms of the way things actually work.
```

Comment:
> The UI used to control a channel.

No:
```
"Fader": The UI control used to adjust the volume for a channel in the mix.   Not “slider” or “volume control”.
```
(The fader _is_ a slider widget.  It _does_ control volume.  But as noted, use "Fader" when you're talking about the UI control for that purpose.  "Slider" alone isn't good enough - to vague.  "Volume control" isn't good enough -- there are other ways to control the volume.)

> Use when "channel" may not be appropriate,

No - it's not a synonym for "channel" in any way.  It only refers to the UI control that's used to adjust volume in the mix.  Nothing else.

> “Each fader has a mute button”

That's wrong.  A fader isn't a button, it's a fader (which is a slider).  It can't be or have a mute button (and it's actually a mute checkbox - anyone looking for a mute button will be disappointed).

> "leftmost fader"

That's okay.

> the visual effect of controlling a group of channels is to "make all the faders move together"

Not necessarily.  Not unless you move one of the faders.  If you hit the solo or mute controls, the faders don't get affected.

> But note that "fader" should **not** be used if "channel" would be a more accurate term.

And "fader" is _only ever_ accurate when referring to the UI control that adjusts the channel volume in the mix.

Reply:
> "Slider" alone isn't good enough - far too vague. "Volume control" isn't good enough -- there are other ways to control the volume.)

That's what I mean by "_Not “slider” or “volume control.”_" (that is, do not use those terms). It may be a bit elliptical, but this is the Style Guide.

>  it's not a synonym for "channel" in any way. It only refers to the UI control that's used to adjust volume in the mix. Nothing else.

Yes, but the point of the Guide is to help us make sure we know how to use "fader" and when to use "channel". They are not synonyms, but unless we are going to ban the term "fader" completely, we need some guidance on how to use it.

> A fader isn't a button, it's a fader (which is a slider). It can't be or have a mute button 

OK so choose a better example. The point here is to give some kind of guidance on possible usage. But if somebody needed to refer to the mute button's location, they might need to refer to it in relation to the fader.

This isn't an exact science :-) And it's not the technical doc for what these things are, it's for writers who may need guidance in unknown contexts. No problem with better examples though.

Reply:
> "Fader" - the UI control that adjust the volume.  It should not be called "volume slider" or "volume control", just "fader" (adding the context of the mixer, if needed).

How's that?  It doesn't need to talk about anything else.  It's confusing to talk about anything else.  All it should talk about is the volume in the mix.

Reply:
Sorry, not sure what you're referring to there.

Reply:
> Sorry, not sure what you're quoting there.

Where?

Reply:
The text you're [quoting here](https://github.com/jamulussoftware/jamuluswebsite/pull/1061#discussion_r1826112335) about "adding the context of the mixer". I can't find that - unless I'm losing the plot.

Reply:
Oh I see - sorry. Yes, that wording effectively bans terms other than "fader" :-)

Reply:
No, saying "group the channels" and "mute the channel" is _correct_.  "Fader" and "Channel" are absolutely _not_ interchangeable.

"Fader" _only_ refers to the adjustment of the channel volume:
* "Use the fader to adjust the channel volume level" would be acceptable -- it explains what the fader UI control does.
* "Use the group checkbox to put channels into a group which you can then adjust further together" would be acceptable -- it explains what group UI control does.
* etc

Reply:
Yes, I know :-) I just wanted to be clear about the decision we're making here, but I'm obviously not explaining myself well enough.

To illustrate - the following hypothetical text is technically correct when describing the UI to someone looking at the Jamulus mixer panel without any other knowledge of how Jamulus works:

_"You can adjust the volume of a channel with the fader, and mute or solo that channel using the button under the fader. You can also group the channels together, and when you move one of the faders in that group, the other faders in that group move with it."_

The following is not technically correct, but just as informative from a naive reader's POV:

_"You can adjust the volume of a channel with the fader, and mute or solo that channel using the button under it. You can also group faders together, and when you move one of the faders in that group, the other faders in that group move with it."_

The way style guides usually work is to mandate one term for what the user sees on the screen. But we don't want to do that because we want to stay true to the underlying technical definitions as that might head off problems later on. And fair enough - just as long as we know we are making that decision when the reader may well logically, if in error, conclude that the terms "fader" and "channel" are interchangeable, and wonder why we appear inconsistent. This is why I was wanting to add that we do this to help "educate the reader".

Reply:
Yes - and the reason
> You can also group faders together, 

is not helpful is that you can also mute and solo groups, I think, so it misleads the reader into thinking that it does _not_ apply to mute and solo controls.

Reply:
Exactly so - although it's not relevant to my point about the style guide :-) But no matter. In the interests of merging this, I won't labour that any more.

Reply:
It's relevant to authors to know _why_ a direction is being given.